### PR TITLE
:art: Remove maymay

### LINKED ISSUE #4: Remove excess memes.
The inclusion of memes was great a few years ago, but now it feels forced and like I'm trying to be _the cool dad_.

### PR DESCRIPTION
**Pull request by jameshughes89**
### Related Issues or PRs
Closes #4 

### What
Remove the use of memes in course content. 

### Why
Was funny at some point in time, but now it feels played out and I'm trying to hard to act like a cool kid. 

### Where
Everywhere

### Additional Notes
Some of the images are kept in the repo, because I like them, and maybe they will resurface.

### CODE REVIEW

File: source/outline.rst [hunk 1]
```diff
@@ -1,8 +1,8 @@
-===================================================================
-Computer Science 161 -- Introduction to Programming
-===================================================================
+=====================================================================
```
Comment (twentylemon) [inline source/outline.rst:1]: Changes in this file are unrelated to the main content of this PR. Revert (most of?) these, and open a new PR for them.

Comment (jameshughes89) [reply source/outline.rst:1]: Seems like something went wrong with a merge.

Comment (jameshughes89) [reply source/outline.rst:1]: nerp, branched off branch by accident.
